src/components/CNavigationDrawer.vue

```diff
@@ -12,17 +12,22 @@
         <slot name="branding"></slot>
       </v-list>
 
-      <!-- Navigation Items -->
-      <div class="c-navigation-drawer-items flex-grow-1">
-        <slot></slot>
-      </div>
-
-      <!-- Footer -->
-      <v-fade-transition>
-        <div v-if="slots['footer'] && !rail" class="my-3 text-no-wrap">
-          <slot name="footer"></slot>
+      <div
+        class="d-flex flex-column overflow-x-hidden overflow-y-auto"
+        style="height: 100%"
+      >
```

Comment:
2 questions:
- Is this necessary because of the previous commit or independent?
- Why is the branding slot not included in this div? Can we merge it with the surrounding div?

Reply:
- this is necessary to fix scrolling in the navbar which never worked correctly
- we cannot include it, because I didn't want the branding to scroll away